packages/ckeditor5-clipboard/docs/features/paste-plaintext.md

```diff
@@ -0,0 +1,45 @@
+---
+menu-title: Paste plain text
+category: features-pasting
+order: 30
+---
+
+{@snippet features/build-paste-source}
+
+# Pasting plain text
+
+The plain text pasting feature is provided through the {@link module:clipboard/clipboard~Clipboard} plugin, which is responsible for the integration with the native clipboard and intercepts all native events like `copy`, `cut` or `drop` and handles them on its side. The goal is to not allow the browser to touch the content in the rich text editor which would lead to the browser messing it up.
+
+The plain text pasting itself is further aided by the {@link module:clipboard/clipboard~PastePlainText} plugin. It detects the <kbd>Ctrl</kbd>/<kbd>Cmd</kbd>+<kbd>Shift</kbd>+<kbd>V</kbd> keystroke during the paste and causes the pasted plain text to inherit the styles of the content it was pasted into.
+
+Pasting plain text with a double line break will turn it into a paragraph. This is a change from the previous behavior, when a single line break resulted in creating a paragraph. As of now, a single line break will be turned into a soft break upon pasting.
```

Comment:
This stands out a bit - you have some general information above and then a detail here, and no more details about the feature (not sure if there are any). If we only have this, then move it to a separate section.

It should also be reworded - the text you used is a direct changelog/release blog post copy and it looks strange in the docs. The docs are assumed to contain the latest, most up-to-date version so if you mention that something has changed, add a version where it changed.